metric/metrics_collector.rb

```diff
@@ -11,7 +11,7 @@ class MetricsCollector
   def initialize(options)
     @options  = options
 
-    @ems = Ems.new(ems_options)
+    @ems = Ems.new(options[:ems_id], ems_options)
```

Comment:
Could this just be one of the `ems_options` ?  There is already a different `vim_opts` method for the connection specific keys